pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -91,6 +95,11 @@ func NewController(
 		UpdateFunc: controller.PassNew(impl.Enqueue),
 		DeleteFunc: impl.Enqueue,
 	})
+
+	r.tracker = tracker.New(impl.EnqueueKey, 30*time.Second)
```

Comment:
I added a better way to get the tracker lease as a function of the resync period in #120 which I can update here once this is merged, no point in both of us making the same change

Reply:
nice!! thanks @pivotal-nader-ziada :D if you merge first ill pickup your change, ill leave this at 30 min for now then